css/index.css

```diff
@@ -106,7 +107,7 @@ header .logo {
 }
 
 .cta .cta-text {
-    font-size: 72px;
+    font-size: 7.2rem;
```

Comment:
Great job on converting all of the font-sizes to rem! Love that you made separate commits for the different changes you did.